File: python/ciudades.py
"""agregar lista de ciudades """
from utilidades import *
from typing import List, Tuple

def main():
    #definir vaeriables
    ciudades: List[str] = []
    ciudad: str 
    msj: Tuple[str, ...] = ("primera","segunda", "tercera", "cuarta", "quinta", "sexta") 
    nro_ciudades: int = 6
    encabezado("ingreso de ciudades", 50)
    for i in range(nro_ciudades):
        ciudad = leerTexto(f"ingrese la {msj[i]} ciudad", "ciudad")
        ciudades.append(ciudad)
        
    mostrar_ciudades(ciudades, 'ver ciudades', 50)
    
    ciudades_asc = list(sorted(ciudades))
    mostrar_ciudades(ciudades_asc, 'ciudades ordenadas ascendentemente', 50)
    
    ciudades_des = list(sorted(ciudades, reverse=True))
    mostrar_ciudades(ciudades_des, 'ciudades ordenadas desendentemente', 50)
    
    
def mostrar_ciudades(ciudades: List[str], titulo: str, largo: int) -> None:
    encabezado(titulo, largo)
    for item in ciudades:
        print(f"la ciudad es {item.upper()}")
    
main()

File: JS/src/ciudades.js
/* crear un array de ciudades */
import * as util from "./utilidades.js";

const main = () => {
    //definir variables
    const ciudades = [];
    let ciudad = '';
    const avisos = [
        'primera',
        'segunda',
        'tercera', 
        'cuarta',
        'quinta', 
        'sexta'
    ];
    const nroCiudades = 6;

    util.encabezado("ingreso de ciudades", 50);
    for (let i = 0; i < nroCiudades; i++) {
        ciudad = util.leerTexto(`ingresa la ${avisos[i]} ciudad`, 'ciudad');
        ciudades.push(ciudad);
    }

    verCiudades(ciudades, 'mostrar ciudades', 50);

    ciudades.sort();
    verCiudades(ciudades, 'ciudades ordendas ascendentemente', 50);
    
    ciudades.sort((a,b) => b.localeCompare(a));
    verCiudades(ciudades, 'ciudades ordendas descendentemente', 50);
   

}

const verCiudades = (ciudades, titulo = ' ', largo = 0) => {
    util.encabezado(titulo, largo);
    ciudades.forEach((item) => console.log(`la ciudad es ${item.toUpperCase()}`));

}

main()